recipe-client-addon/lib/EventEmitter.jsm

```diff
@@ -1,17 +1,16 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-// This file is meant to run inside action sandboxes
-
 "use strict";
 
+const {utils: Cu} = Components;
+Cu.import("resource://shield-recipe-client/lib/LogManager.jsm");
```

Comment:
My brain is broken and so I don't remember this thing. It doesn't seem to be in this diff. I can't find it in mozilla-central. It also doesn't seem to be in the list of modules to unload in bootstrap.js - I'm assuming that's just an oversight and this got added elsewhere? Or something?